client/my-sites/site-settings/navigation.jsx

```diff
@@ -86,21 +85,11 @@ export default React.createClass( {
 							{ strings.discussion }
 					</NavItem>
 
-					{
-						( ( ! site.jetpack && config.isEnabled( 'manage/plans' ) ) ||
-							( site.jetpack && config.isEnabled( 'jetpack/google-analytics' ) ) ) &&
```

Comment:
Are we not concerned any more about Jetpack? it seems like this logic check disappears but I didn't see any obvious move. I thought this stuff didn't work for Jetpack sites